src/Controllers/AboutController.ts

```diff
@@ -86,252 +80,50 @@ module StreamStats.Controllers {
-
+            
             //check if this state/region is enabled in appConfig.js
-            configuration.regions.forEach((value, index) => {
-
+            configuration.regions.forEach((value) => {
                 //find this state/region
                 if (value.Name === regionID) {
                     if (!value.regionEnabled) {
-                         this.regionArticle = '<div class="wim-alert">StreamStats has not been developed for <strong>' + value.Name + '</strong>.  Please contact the <a href="mailto:support@streamstats.freshdesk.com">streamstats team</a> if you would like StreamStats enabled for this State/Region.</div>';
+                        this.regionArticle = '<div class="wim-alert">StreamStats has not been developed for <strong>' + value.Name + '</strong>.  Please contact the <a href="mailto:support@streamstats.freshdesk.com">streamstats team</a> if you would like StreamStats enabled for this State/Region.</div>';
                     }
-
-                    //otherwise get region help article
+                    //otherwise get region help url
                     else {
-                        //clear article
-                        this.regionArticle = '<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>';
-
-                        this.Execute(request).then(
-                            (response: any) => {
-
-                                response.data.folder.articles.forEach((article) => {
-                                    if (article.title == regionID) {
-                                        //console.log("Help article found for : ", regionID);
-                                        if (window.location.host.indexOf('test.streamstats.usgs.gov') > -1) {
-                                            this.regionArticle = article.description;
-                                        } else if (article.status == 2) {
-                                            this.regionArticle = article.description;
-                                        }
-                                        return;
-                                    }
-                                });
-
-                            }, (error) => {
-                                //sm when error
-                            }).finally(() => {
-
-                            });
+                        if (value.URL == null) { // Doesnt have state webpage
+                            this.regionArticle = '<div class="wim-alert">There is currently no information page for <strong>' + value.Name + '</strong>.  Please contact the <a href="mailto:support@streamstats.freshdesk.com">streamstats team</a> with any questions.</div>';
+                        } else { // Has state webpage
+                            this.regionArticle = "<p>To find information about the data used for <strong>"+ value.Name + "</strong>, see our <a href="+this.regionURL+">"+ value.Name + " information </a> page.</p>";

```

Comment:
When I click this link, it just goes to https://www.usgs.gov/streamstats/state-and-region-based-info rather than https://www.usgs.gov/streamstats/mississippi-streamstats 


![image](https://user-images.githubusercontent.com/40237491/218540759-114b9657-3b31-4fb0-a815-e5150d6a6e45.png)